indico/modules/events/contributions/models/fields.py

```diff
@@ -44,6 +44,10 @@ class ContributionField(db.Model):
         index=True,
         nullable=False
     )
+    legacy_id = db.Column(
+        db.String,
+        nullable=True
+    )
```

Comment:
and for abstract you could simply rename the `legacy_id` to `friendly_id` since that's pretty much what it is